src/rpc/rpc_reward_api.py

```diff
@@ -52,8 +52,18 @@ def get_rewards_for_cycle_map(self, cycle, rewards_type):
             reward_data["delegators_nb"] = len(reward_data["delegators"])
 
             # Get last block in cycle where rewards are unfrozen
-            level_of_last_block_in_unfreeze_cycle = (cycle + self.preserved_cycles + 1) * self.blocks_per_cycle
-            level_of_first_block_in_preserved_cycles = (cycle - self.preserved_cycles) * self.blocks_per_cycle + 1
+            if cycle < 394:
```

Comment:
All numbers should be at the top in constants

Reply:
Done